src/hotspot/share/runtime/lockStack.inline.hpp

```diff
@@ -38,6 +39,8 @@
 
 inline int LockStack::to_index(uint32_t offset) {
   assert(is_aligned(offset, oopSize), "Bad alignment: %u", offset);
+  assert((offset <= end_offset()), "lockstack overflow: offset %d end_offset %d", offset, end_offset());
+  assert((offset >= start_offset()), "lockstack underflow: offset %d end_offset %d", offset, start_offset());
```

Comment:
nit typo: s/end_offset/start_offset/

Reply:
Good. That is a copy-paste-typo. So two typo fixes coming up.